### PR TITLE
Add Azure vm_size support

### PR DESCRIPTION
**Pull request by TheoBrigitte**
Provide the new `--azure-vm-size` to `gsctl create cluster`
Show `Worker vm size: <vm size>` on `gsctl show cluster`

Require: https://github.com/giantswarm/gsclientgen/pull/24

### CODE REVIEW

**Comment by marians** [comment]
If CI fails, then likely something is not working :)

I normally

- use the dedicated `gsctlientgen` branch instead of master in `Gopkg.toml`
- get reviews/approvals
- merge the gsclientgen PR
- change gsctl back to use gsclientgen master
- merge the gsctl PR

File: commands/show_cluster.go [hunk 1]
```diff
@@ -218,8 +218,14 @@ func showClusterRunOutput(cmd *cobra.Command, cmdLineArgs []string) {
...
 	// This assumes all nodes use the same instance type.
-	if len(clusterDetails.Workers) > 0 && clusterDetails.Workers[0].Aws.InstanceType != "" {
-		output = append(output, color.YellowString("Worker instance type:")+"|"+clusterDetails.Workers[0].Aws.InstanceType)
+	if len(clusterDetails.Workers) > 0 {
+		if clusterDetails.Workers[0].Aws.InstanceType != "" {
+			output = append(output, color.YellowString("Worker instance type:")+"|"+clusterDetails.Workers[0].Aws.InstanceType)
+		}
+
+		if clusterDetails.Workers[0].Azure.VmSize != "" {
+			output = append(output, color.YellowString("Worker vm size:")+"|"+clusterDetails.Workers[0].Azure.VmSize)
```
Comment (marians) [inline commands/show_cluster.go]: I'd prefer `Worker VM size:`

**Comment by TheoBrigitte** [comment]
I was just thinking to make it the other way around:
- get review/approval for `gsctlientgen`
- merge `gsctlientgen`
- update `gsctl` vendor to latest gsctlientgen:master`
- get review/approval for `gsctl`

but this is impeding `gsctl` approval, I'll update this

**Comment by marians** [comment]
I like having CI tests pass in gsctl before merging. :)

**Comment by TheoBrigitte** [comment]
Towards https://github.com/giantswarm/giantswarm/issues/2454

**Pull request merged**